mocked-api/server.js

```diff
@@ -0,0 +1,22 @@
+const jsonServer = require('json-server');
+require('dotenv').config();
+const server = jsonServer.create();
+const router = jsonServer.router('mocked-api/db.json');
+const middlewares = jsonServer.defaults();
+
+const isAuthorized = (req) => {
+  return req.headers.authorization === `Bearer ${process.env.VITE_API_TOKEN}`;
```

Comment:
Why did you decide to add an authorization header to requests?

Reply:
I have not written apps without any kind of authentication or authorization (sometimes it was internal tokens, sometimes Authorization code flow, Implicite flow or other). So it looked more "natural" to have anything.

Reply:
@bednarskimichal Authorization headers are now depreacted nowdays we pass auth headers in request footer